src/api/cloud/project/lab/cloud-project-lab.v6.service.js

```diff
@@ -0,0 +1,25 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectLabV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/lab/:labId', {
```

Comment:
`$resource` can be directly returned then.

Reply:
Done + cache added.